src/components/util/util.js

```diff
@@ -15,6 +15,26 @@ export function getFullName(user) {
   return `${lastName}${firstName ? ', ' : ' '}${firstName}${middleName ? ' ' : ''}${middleName}`;
 }
 
+export const formatActionDescription = (action) => {
+  return action.typeAction +
+    (action.paymentMethod
+      ? ('-' + action.paymentMethod)
+      : ' '
+    );
+};
+
+export const formatCurrencyAmount = (amount = 0) => parseFloat(amount).toFixed(2);
```

Comment:
ui-tenant-settings allows tenants to set their [primary currency](https://folio-snapshot.dev.folio.orgc), "for currency symbol to display". Shouldn't this act just like `formatDateAndTime()` below? Pull `formatNumber` out of `intl` the same way you get [`formatTime`](https://github.com/folio-org/ui-users/pull/1611/files#diff-7c8a7254ac9b2f95f8a27158b661f580b7d43f49c6228f04436b4aa5e2acf6c0R42) and the implement this like 
```
export const formatCurrencyAmount = (amount, formatter) => formatter(amount, { style: 'currency' });
```
Apologies if this is out of scope; feel free to ignore it if so. Er, feel free to tell me we need to file a bunch of Jiras to correctly handle currency display if so.

Reply:
@zburke that's a good idea to create, not a bunch I suppose, but single separate ticker for `ui-users`. There are lots of places of currency amount.

Reply:
Filed https://issues.folio.org/browse/UIU-2026